prepare_shop/action.yaml

```diff
@@ -242,11 +196,11 @@ runs:
           -H 'Accept: application/vnd.github.raw+json' \
           '${{ inputs.compilation_url }}'
 
-    - name: Clone the shop  (${{ inputs.git_shop_ref }})
-      if: ${{ inputs.compilation_url == '' && (inputs.github_event_name != 'pull_request' || ( github.repository != 'OXID-eSales/oxideshop_ce')) }}
+    - name: 'Clone ${{ inputs.git_repository }} (${{ inputs.git_ref }})'
+      if: ${{ inputs.compilation_url == '' && inputs.github_event_name != 'pull_request' }}
       shell: bash
       run: |
-        # prepare_shop: Clone the shop
+        # prepare_shop: Clone ${{ inputs.git_repository }} (${{ inputs.git_ref }})
```

Comment:
The "name" and the comment should match. The comment is actually a workaround for github not showing step names inside actions properly and this helps us finding the right step in the code